### PR TITLE
Release v3.6.1

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

**Review by mkondratek — APPROVED** [review]

File: frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties [hunk 1]
```diff
@@ -19,14 +19,14 @@
...
+action.GitMachete.description.disabled.another-actions-ongoing=Action disabled because of the ongoing {0}
 action.GitMachete.get-git-config.task-title=Getting git config value\u2026
 action.GitMachete.set-git-config.task-title=Setting git config value\u2026
 
 
 # Backgroundables
 
-action.GitMachete.CheckRemoteBranchBackgroundable.task-title=Checking remote branch\u2026
-action.GitMachete.CheckRemoteBranchBackgroundable.notification.fail.text=The remote branch ''{0}'' does not exist, cannot pull anymore.
+action.GitMachete.Pull.notification.remote-branch-missing.text=The remote branch ''{0}'' does not exist, cannot pull anymore.
```
Comment (PawelLipski) [inline frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties:29]: Reorder, all `Pull` keys together

File: frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/highlighting/MacheteAnnotator.java [hunk 2]
```diff
@@ -139,17 +135,15 @@ private String getParentBranchName(IBranchLayoutReader branchLayoutReader, PsiFi
     } catch (BranchLayoutException e) { // might appear if branchLayout has inconsistent indentation characters or file is inaccessible
       return "";
     }
-    BranchLayoutEntry parentEntry;
-    try {
-      parentEntry = Objects.requireNonNull(branchLayout.getEntryByName(branchName)).getParent();
-    } catch (NullPointerException e) { // might appear if saveDocument(file) has not completed yet
+    val entry = branchLayout.getEntryByName(branchName);
+    if (entry == null) { // might happen if saveDocument(file) has not completed yet
       return "";
```
Comment (PawelLipski) [inline frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/highlighting/MacheteAnnotator.java:140]: Rather return `null`, for cleanliness

Comment (PawelLipski) [reply frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/highlighting/MacheteAnnotator.java:140]: Done on #1588

File: frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties (hunk 1 above)
Comment (PawelLipski) [reply frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties:29]: #1588

**Pull request merged**